master/internal/config/config.go

```diff
@@ -155,6 +155,8 @@ type Config struct {
 }
 
 // GetMasterConfig returns reference to the master config singleton.
+// This will not reflect the ephimeral changes the user might make to
```

Comment:
can you file a ticket for this one and leave it here?

Reply:
Done.